### PR TITLE
DOCS-946: Add new attribute `pointcloud_file_path` to image file camera

### PR DESCRIPTION
**Pull request by sguequierre**
* Adds the new optional attribute `pointcloud_file_path` to the Image File camera configuration docs page: in the Raw JSON & attributes table
* Changes wording for `file_path` attributes to local system re. SME usage tip (needs review)

### CODE REVIEW

File: docs/components/camera/image-file.md [hunk 1]
```diff
@@ -66,10 +67,11 @@ The following attributes are available for `image_file` cameras:
...
 | `debug` | boolean | Optional | Enables the debug outputs from the camera if `true`. <br> Default: `false` |
-| `color_image_file_path` | string | Optional | The file path to the color image. |
-| `depth_image_file_path` | string | Optional | The file path to the depth image. |
+| `color_image_file_path` | string | Optional | The file path to the color image on your local system. |
+| `depth_image_file_path` | string | Optional | The file path to the depth image on your local system. |
+| `pointcloud_file_path` | string | Optional | The file path to the point cloud file on your local system. You must specify `depth_image_file_path` to use `pointcloud_file_path`. |
 
-You must specify `color_image_file_path` or `depth_image_file_path`.
+You must specify either `color_image_file_path` or `depth_image_file_path`.
```
Comment (sguequierre) [inline docs/components/camera/image-file.md:74]:
```suggestion
You must specify `color_image_file_path` or `depth_image_file_path`.
```

**Review by andf-viam — APPROVED** [review]

File: docs/components/camera/image-file.md [hunk 2]
```diff
@@ -66,8 +67,9 @@ The following attributes are available for `image_file` cameras:
...
 | `distortion_parameters` | object | Optional | Modified Brown-Conrady parameters used to correct for distortions caused by the shape of the camera lens: <ul> <li> <code>rk1</code>: The radial distortion x. </li> <li> <code>rk2</code>: The radial distortion y. </li> <li> <code>rk3</code>: The radial distortion z. </li> <li> <code>tp1</code>: The tangential distortion x. </li> <li> <code>tp2</code>: The tangential distortion y. </li> </ul> |
 | `debug` | boolean | Optional | Enables the debug outputs from the camera if `true`. <br> Default: `false` |
-| `color_image_file_path` | string | Optional | The file path to the color image. |
-| `depth_image_file_path` | string | Optional | The file path to the depth image. |
+| `color_image_file_path` | string | Optional | The file path to the color image on your local system. |
+| `depth_image_file_path` | string | Optional | The file path to the depth image on your local system. |
+| `pointcloud_file_path` | string | Optional | The file path to the point cloud file on your local system. You must specify `depth_image_file_path` to use `pointcloud_file_path`. |
```
Comment (bhaney) [inline docs/components/camera/image-file.md:72]: It's actually just necessary to have one of the three `file_path` set. You can also just have `pointcloud_file_path` without the other two files paths set.

**Review by bhaney — CHANGES_REQUESTED** [review]
just having the pointcloud_file_path and nothing else set is also valid now

File: docs/components/camera/image-file.md (hunk 2 above)
Comment (sguequierre) [reply docs/components/camera/image-file.md:72]: Ok, cool! In that case I am also going to remove this usage note at the top of the page: 

```
If you provide a depth image, as well as the intrinsic parameters of the camera that took that depth image, then the camera will also provide a point cloud.
```
Assuming this is now inaccurate and this attribute simplifies usage?

Comment (sguequierre) [inline docs/components/camera/image-file.md:72]:
```suggestion
| `pointcloud_file_path` | string | Optional | The file path to the point cloud file on your local system. |
```

File: docs/components/camera/image-file.md [hunk 3]
```diff
@@ -51,7 +51,8 @@ Edit and fill in the attributes as applicable.
         },
         "debug": <boolean>,
         "color_image_file_path": "<your-file-path>",
-        "depth_image_file_path": "<your-file-path>"
+        "depth_image_file_path": "<your-file-path>",
+        "pointcloud_file_path": "<your-file-path-to-pcd-file>",
```
Comment (sguequierre) [inline docs/components/camera/image-file.md:55]:
```suggestion
        "pointcloud_file_path": "<your-file-path>",
```

File: docs/components/camera/image-file.md [hunk 4]
```diff
@@ -66,8 +66,9 @@ The following attributes are available for `image_file` cameras:
...
 | `distortion_parameters` | object | Optional | Modified Brown-Conrady parameters used to correct for distortions caused by the shape of the camera lens: <ul> <li> <code>rk1</code>: The radial distortion x. </li> <li> <code>rk2</code>: The radial distortion y. </li> <li> <code>rk3</code>: The radial distortion z. </li> <li> <code>tp1</code>: The tangential distortion x. </li> <li> <code>tp2</code>: The tangential distortion y. </li> </ul> |
 | `debug` | boolean | Optional | Enables the debug outputs from the camera if `true`. <br> Default: `false` |
-| `color_image_file_path` | string | Optional | The file path to the color image. |
-| `depth_image_file_path` | string | Optional | The file path to the depth image. |
+| `color_image_file_path` | string | Optional | The file path to the color image on your local system. |
+| `depth_image_file_path` | string | Optional | The file path to the depth image on your local system. |
+| `pointcloud_file_path` | string | Optional | The file path to the point cloud file on your local system. |
 
 You must specify `color_image_file_path` or `depth_image_file_path`.
```
Comment (bhaney) [inline docs/components/camera/image-file.md:73]: you also have the option of specifying the `pointcloud_file_path` now as well.

File: docs/components/camera/image-file.md [hunk 5]
```diff
@@ -9,8 +9,7 @@ tags: ["camera", "components"]
 # SMEs: Bijan, vision team
 ---
 
-An `image_file` camera gets color and depth images frames from a file path.
-If you provide a depth image, as well as the intrinsic parameters of the camera that took that depth image, then the camera will also provide a point cloud.
```
Comment (bhaney) [inline docs/components/camera/image-file.md:13]: This second line is still true. There are two ways of producing point clouds now:

1. provide a path to a pcd file (new way)
2. provide a path to a depth image, as well as the intrinsic parameters of the camera that took the depth image.

If you do both, the pcd file takes precedence

**Review by bhaney — CHANGES_REQUESTED** [review]

File: docs/components/camera/image-file.md (hunk 5 above)
Comment (sguequierre) [reply docs/components/camera/image-file.md:13]: Ok, thank you for explaining this! Will update and request for re-review once addressed

Comment (sguequierre) [reply docs/components/camera/image-file.md:13]: Think this is addressed now but I moved the usage tip below, lmk if any issues

**Review by bhaney — APPROVED** [review]
Looks good! Thanks

**Pull request merged**